esp/src/eclwatch/HPCCPlatformWidget.js

```diff
@@ -415,7 +416,18 @@ define([
         _onAboutClose: function (evt) {
             this.aboutDialog.hide();
         },
-
+		
+        _onLogout: function (evt) {
+            xhr("esp/logout", {
+                method: "post"
+            }).then(function(data){
+                if (data){
+                    document.cookie = "ESPSessionID" + location.port + " = '' "; "expires=Thu, 01 Jan 1970 00:00:00 GMT"; // or -1
+                    window.location.reload();
```

Comment:
@miguelvazq @kunalaswani 
Shouldn't ESPs response from this server call be responsible for killing the cookie session?  
Since ESP put the cookie in, it should also clear it I would have thought?